packages/esm-ward-app/src/ward-view-header/admission-requests.component.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Movement } from '@carbon/react/icons';
+import styles from './admission-requests.scss';
+import { launchWorkspace } from '@openmrs/esm-framework';
+
+const AdmissionRequests = () => {
```

Comment:
Maybe name it `AdmissionRequestBar` and/or put a comment that it's for the bar + button at the top right corner?